ext/json11/CMakeLists.txt

```diff
@@ -4,6 +4,38 @@
 #  Boost Software License, Version 1.0. (See accompanying file
 #  LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
 
-project(json11)
+project(json11
+	VERSION 0.1)
 
-add_library(${PROJECT_NAME} json11.hpp json11.cpp)
+add_library(${PROJECT_NAME} STATIC json11.hpp json11.cpp)
+
+# Library installation directory
+if(NOT DEFINED CMAKE_INSTALL_LIBDIR)
+    set(CMAKE_INSTALL_LIBDIR lib)
+endif(NOT DEFINED CMAKE_INSTALL_LIBDIR)
+set(libdir ${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR})
+
+set_target_properties(json11 PROPERTIES DEBUG_POSTFIX -d FOLDER json11)
```

Comment:
Here and below in this file use `${PROJECT_NAME}` instead of harcoded `json11`.

Reply:
OK.